rare/components/tabs/games/game_info/game_info.py

```diff
@@ -143,47 +143,52 @@ def verify(self):
             return
         self.verify_widget.setCurrentIndex(1)
         verify_worker = VerifyWorker(self.game.app_name)
-        verify_worker.signals.status.connect(self.verify_statistics)
-        verify_worker.signals.summary.connect(self.finish_verify)
+        verify_worker.signals.status.connect(self.verify_status)
+        verify_worker.signals.result.connect(self.verify_result)
+        verify_worker.signals.error.connect(self.verify_error)
         self.verify_progress.setValue(0)
         self.verify_threads[self.game.app_name] = verify_worker
         self.verify_pool.start(verify_worker)
         self.move_button.setEnabled(False)
 
-    def verify_statistics(self, num, total, app_name):
+    @pyqtSlot(str, str)
+    def verify_error(self, app_name, message):
+        pass
```

Comment:
I think it was a TODO iirc, since we can't/shouldn't show dialogs in threads, a signal could be passed. I didn't have the implementation in mind yet because I didn't have the `LgndrIndirectStatus` implemented yet.